metar/Station.py

```diff
@@ -20,8 +21,9 @@ def __init__( self, id, city=None, state=None, country=None, latitude=None, long
       self.name = "%s, %s" % (self.city, self.state)
     else:
       self.name = self.city
-    
-station_file_name = "nsd_cccc.txt"
+
+current_dir = os.path.split(__file__)[0]
```

Comment:
A slightly better way to do this is 

current_dir = os.path.dirname(__file__)

Reply:
thanks, I adapted my patch.